CONTRIBUTING.md

```diff
@@ -1,128 +1,82 @@
-
-* Our [mailing list](https://www.ietf.org/mailman/listinfo/quic) is used for most communication, including notifications of meetings, new drafts, consensus calls and other business, as well as issue discussion.
-
-* We also discuss specific issues on the appropriate issues list in [Github](https://github.com/quicwg/). If you don't want to use Github to follow these discussions, you can subscribe to the [issue announce list](https://www.ietf.org/mailman/listinfo/quic-issues).
-
-To be active in the Working Group, you can participate in any of these places. Most activity takes
-place on the mailing list, but if you just want to comment on and raise issues, that's fine too.
-
-
-## Raising Issues
-
-We use our [Github](https://github.com/quicwg/) issues lists to track items for discussion and
-their resolution.
-
-Before filing a new issue, please consider a few things:
-
-* Issues should be just that; issues with our deliverables, **not proposals, questions or support requests**.
-* Please review the issues list to make sure that you aren't filing a duplicate.
-* If you're not sure how to phrase your issue, please ask on the [mailing list](https://www.ietf.org/mailman/listinfo/quic).
-
-Issues can also be raised on the [Working Group mailing
-list](https://www.ietf.org/mailman/listinfo/quic) by clearly marking them as such (e.g., "New
-Issue" in the `Subject:` line).
-
-Be aware that issues might be rephrased, changed in scope, or combined with others, so that the
-group can focus its efforts. If you feel that such a change loses an important part of your
-original issue, please bring it up, either in comments or on the list.
-
-Off-topic and duplicate issues will be closed without discussion. Note that comments on individual
-commits will only be responded to with best effort, and may not be seen.
-
-
-## Resolving Issues
-
-Issues will be labeled by the Chairs as either `editorial` or `design`:
-
-* **Design** issues require discussion and consensus in the Working Group. This discussion can happen both in the issue and on the [Working Group mailing list](https://www.ietf.org/mailman/listinfo/quic), as outlined below.
-
-* **Editorial** issues can be dealt with by the editor(s) without consensus or notification. Typically, any discussion will take place on the issue itself.
-
-The open design issues in the issues list are those that we are currently discussing, or plan to discuss. They can be discussed on the mailing list or the issue itself.
-
-We're currently using two different processes for issue resolution, depending on draft maturity.
+Any change will be subject to careful review and discussion, which might involve
+the editors, chairs, Working Group, and our Area Director.
 
```

Comment:
Maybe add:
```suggestion
After this review and discussion, errata may need to be filed for the published RFCs,
or the Working Group may otherwise address such valid issues.
```